feeds.conf

```diff
@@ -1,4 +1,4 @@
-src-git packages https://git.openwrt.org/feed/packages.git^f5cb61cee5ef79903ee3c093fb31974c13e78ea5
-src-git luci https://git.openwrt.org/project/luci.git;openwrt-19.07
-src-git routing https://git.openwrt.org/feed/routing.git^9b42e24a54f03ebb6f58224b49036e8f739b175f
-src-git telephony https://git.openwrt.org/feed/telephony.git;openwrt-19.07
+src-git packages https://git.openwrt.org/feed/packages.git^09da83968ef0846cd1b13bfa1b91c33a1f9985bb
+src-git luci https://git.openwrt.org/project/luci.git;openwrt-22.03
```

Comment:
der relevante patch ist auch in 19.07 ich find jetzt auf 22.03 zu gehen echt wild. sows muss man gut vorbereiten und testen. bitte 19.07 - gillt natürlich auch für hashes